hawkbit-ui/src/main/java/org/eclipse/hawkbit/ui/rollout/window/components/AdvancedGroupRow.java

```diff
@@ -34,6 +35,8 @@
  * Advance group row component
  */
 public class AdvancedGroupRow {
+    
+    private static final String DIST_CHECKBOX_STYLE = "dist-checkbox-style";
```

Comment:
Can be moved to SPUIStyleDefinitions

Reply:
also refactored some other classes to reuse it